components/list/list-item-drag-drop-mixin.js

```diff
@@ -37,7 +38,7 @@ const isDragSupported = () => {
 	return typeof el.ondragenter === 'function';
 };
 
-export const ListItemDragDropMixin = superclass => class extends superclass {
+export const ListItemDragDropMixin = superclass => class extends LocalizeCoreElement(superclass) {
```

Comment:
This line breaks `list-item-generic-layout` tests >_> WHY.

Reply:
This is definitely a tomorrow problem, but I _believe_ what's happening in the tests is that the `shadowRoot` of the `list-item`s are not loaded until the locale resources load, which results in a bunch of `null` shadow roots when we try to query the individual list items.

We need a way to wait for the locale resources before attempting to query the shadow root.
```js
it('enables grid on list-item-generic-layout elements', async() => {
	const el = await fixture(normalFixture); // this waits for the list but not necessarily the list-items in it
	await elementUpdated(el._getItems()[0]); // this works sometimes, but not all the time
	el._getItems().forEach(item => {
		expect(item.shadowRoot.querySelector('d2l-list-item-generic-layout').gridActive).to.be.true; // these shadowRoots are null if the items haven't loaded completely
	});
});
```

Reply:
We could also keep the LocalizeCoreElement in the `list-item-drag-handle` and we pass down an object with the position and the count instead of the string?